lib/nfs_common.pm

```diff
@@ -183,5 +183,85 @@ sub yast2_server_initial {
     } while (not match_has_tag('nfs-config'));
 }
 
+sub install_service {
+    # Make sure packages are installed
+    zypper_call 'in yast2-nfs-server nfs-kernel-server', timeout => 480, exitcode => [0, 106, 107];
```

Comment:
`yast2-nfs-server` should take care of installing `nfs-kernel-server`. Now this isn't tested anymore.

Reply:
I just move the original code to here, not change anything. If you delete nfs-kernel-server, then you will encounter issue http://openqa.suse.de/tests/3264073#step/yast2_nfs_server/4 . 
So i would not change this.